dbms/src/Flash/Coprocessor/DAGContext.h

```diff
@@ -447,6 +452,11 @@ class DAGContext
     // - Stream: execute with block input stream
     // - Pipeline: execute with pipeline model
     ExecutionMode execution_mode = ExecutionMode::None;
+
+    // It's the session id between client and tidb
```

Comment:
Is comment expected? `between client and tidb` or `between client and tiflash`

Reply:
> Is comment expected? `between client and tidb` or `between client and tiflash`

It's expected as tidb is the session id is allocated in tidb.

Reply:
ok, `mysql client between mysql client and tidb` will be clearer.

Reply:
> ok, `session id between mysql client and tidb` will be clearer.

okk